gensim/models/lda_dispatcher.py

```diff
@@ -15,14 +15,21 @@
 
 
 from __future__ import with_statement
-import os, sys, logging, threading, time
+import argparse
```

Comment:
I checked, this triggered only on importing `lda_dispatcher.py` or `lda_worker.py`. 
Backport for argparse in `setup.py` for python < 2.7 ([proof](https://github.com/RaRe-Technologies/gensim/blob/master/setup.py#L119))